### DISCUSSION TITLE
where is `splitrule = 'conserve'`?

### DISCUSSION THREAD

**Discussion by EQUIWDH** [kogalur/randomForestSRC#365] [General]
Excuse me, I am studying random survival forest recently. My question is that the R package `randomForestSRC` seem not to provide the 'conservation of event' split rule. Do U guys remove it in earlier version? Because I just read the paper whose doi is 10.1214/08-AOAS169
 and the R-news ISSN 1609-3631 shows that this package should include the conservation of event rule. Thank U!
![屏幕截图 2023-02-19 185250](https://user-images.githubusercontent.com/101512154/219943523-0cce4e48-9660-404c-864c-3ad6a9ece967.png)

**Comment by ishwaran** [comment]
That split rule is no longer used and has not been used for many years in our package.  The default splitting rule for survival is `logrank`
